### ISSUE TITLE
v0.44 - AIRSPY not working on Rasbian

### ISSUE THREAD

**Issue by anttinousiainen** [jvde-github/AIS-catcher#76]
Any pointers on what should I try to get Airspy to work on Rasbian.

- cloned the latest, same versions from github on Windows and Rasbian
- installed Airspy libraries on Windows and Rasbian
- compiled on Windows and Rasbian, both just fine

- AIS-catcher -l on Rasbian lists SDR support for both RTLSDR AIRSPY (and others)
- AIS-catcher -l on Windows lists SDR support for both RTLSDR AIRSPY (and others)
- AIS-catcher -L on Rasbian only lists RTLSDR device

- dmesg on Rasbian shows both normally

- rtlsdr receives fine on Rasbian
- Airspy receives fine on WIndows

It seems the ais-catcher is somehow not recognizing airspy on rasbian.

So any ideas on how to get Airspy working on Rasbian as well, any pointers what should I try next?

I will try some other software next myself, then maybe some older version of ais-catcher.

**Comment by anttinousiainen** [comment]
printout from 

WIndows:

.\AIS-catcher.exe -L
AIS-catcher (build Jan 31 2023) v0.44
(C) Copyright 2021-2023 jvde-github and other contributors
This is free software; see the source for copying conditions.There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
SDR support: RTLSDR AIRSPY AIRSPYHF+ RTLTCP SPYSERVER ZMQ HACKRF
Other support: SOXR CURL ZLIB RTLSDR-BIASTEE RTLSDR-TUNERBW

Rasbian:

./AIS-catcher -L
AIS-catcher (build Feb  9 2023) v0.44-86-gca18b31
(C) Copyright 2021-2023 jvde-github and other contributors
This is free software; see the source for copying conditions.There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
SDR support: RTLSDR AIRSPY RTLTCP SPYSERVER
Other support: CURL ZLIB LIBSAMPLERATE

I notice at least on Windows it seems I am running an older version than the one I built. Have to investigate.

**Comment by anttinousiainen** [comment]
Got it working by reinstalling airspy dev libraries and recompiling aiscatcher.

Also noticed by default rasbian now requires anything airspy related to be run under sudo. So it might have been just a permission glitch. It is strange, because same hw worked before together.

Anyhow, this is now solved.

tldr: recompile everything and run under sudo. Now everything works.

**Comment by anttinousiainen** [comment]
Solved.

**Comment by anttinousiainen** [comment]
> If I encounter the same issue I will add some comments to the build instruction.

I will let you know if there is any more issues, I will rebuild a few times as I need to install a few new Rasbians.

BTW, speaking of Airspy, could it receive all four channels (A, B, C and D) at the same time, as it should have enough bandwidth (10Mhz) or is there some technical issue stopping that? It seems channels C and D see quite a lot of activity, and with those you indeed see ships which are further away, ie. get more warning if onboard a boat.

**Comment by anttinousiainen** [comment]
> Other idea might be to pick up DSC and voice as well in this band. Are you using/interested in that?

Both would be super interesting. DSC on its own right, but voice especially. Could scan and listen to quite many voice channels at the same time? Would need to detect active channel ie. with someone transmitting, some sort of squelch, and deal with simultaneous transmissions, but yes!

**Comment by anttinousiainen** [comment]
> Have not looked at it yet and need to grab some recordings first on DSC and audio but technically feasible to listen to it simultaneously on an airspy.

This would be excellent, though then you need to change the name to marine-vhf-catcher :)

No but seriously this could be very valuable tool for onboard and on shore. Could monitor everything happening on the marine band at the same time, AIS, DSC and audio. This could be a valuable tool for online situational awareness, but for port operations etc also for logging what happens for later replay.

In the future there will be VDES and dPMR as well, but those can be added when the time comes.
